requirements.txt

```diff
@@ -0,0 +1,7 @@
+astropy==4.1
```

Comment:
Are you only using this for coordinate conversion? we could make our own functions very easily

Reply:
I'm guessing we add packages to this as we go? its a living doc right?

Reply:
@jonesbmj  will take care of making our own conversion functions in another branch